src/test/java/gov/healthit/chpl/aqa/asserts/ManageSurveillanceAsserts.java

```diff
@@ -0,0 +1,31 @@
+package gov.healthit.chpl.aqa.asserts;
+
+import static org.testng.Assert.assertTrue;
+
+import org.openqa.selenium.NoSuchElementException;
+
+import cucumber.api.java.en.Then;
+import gov.healthit.chpl.aqa.pageObjects.DpManagementPage;
+import gov.healthit.chpl.aqa.stepDefinitions.Base;
+import gov.healthit.chpl.aqa.stepDefinitions.Hooks;
+/**
+ * Class ManageSurveillanceAsserts definition.
+ */
+public class ManageSurveillanceAsserts extends Base {
+
+    /**
+     * Verify surveillance search results load by validating CHPL ID in results.
+     * @param chplId is chplId to look up
+     * @throws Exception if results did not load and chplId was not found
+     */
+    @Then("^I see the surveillance results for \"([^\"]*)\"$")
+    public void testSurveillanceResultsAsExpected(final String chplId) throws Exception {
+        try {
+            String actualString = DpManagementPage.manageProductSurveillance(getDriver()).getText();
```

Comment:
Going to approve this, but this test is using the DpManagementPage, which is the page that's going to go away completely as soon as the adminNav flag goes away. It would be good to migrate any tests that we're changing / move any that we're creating, to use the new nav pages as we can

Reply:
I'll start working on renaming existing/adding new pages to align with new nav soon